dbusmock/mockobject.py

```diff
@@ -322,6 +322,23 @@ def Set(self, interface_name: str, property_name: str, value: Any, *_, **__) ->
             [interface_name, dbus.Dictionary({property_name: value}, signature="sv"), dbus.Array([], signature="s")],
         )
 
+    def add_object(self, mock_class: Type[Self], path: str, interface: str, properties: PropsType, methods: List[MethodType]) -> Self:
```

Comment:
Having another public method on this class feels a bit confusing. Why not simply add a kwarg to the existing AddObject? If that somehow interferes with the decorator, then can you please make it private? Like `_add_object_with_class`?

Reply:
I did add it as kwargs now but the arguments `mock_class` and `mock_data` are only useful when the function is called directly and not via dbus.